### ISSUE TITLE
Unity 2018.3.0 and PostProcessing 2.1.2

### ISSUE THREAD

**Issue by FlaxenFlash** [jean-moreno/EdgeDetect-PostProcessingUnity#11]
The effect doesn't seem to work at all after updating Unity and the post processing stack. It doesn't edge highlight at all with any of the effect sorting options and in some of them it just blacks out the entire screen.

**Comment by Tackleberry23** [comment]
Same here bro, same here.

**Comment by Rockylars** [comment]
I've been waiting with updating to 2018.3, is there any example you could give @FlaxenFlash and @Tackleberry23, what are your configs for it?

Would be nice to see if my setup would work before i ruin it entirely.

![image](https://user-images.githubusercontent.com/21218585/50654813-53f74f00-0f8e-11e9-9cb0-8900c4743485.png)
I use a very simple setup to keep depth visible.

**Comment by FlaxenFlash** [comment]
The main issue is that some of the post processing stack shader include files have moved again, similar to this issue https://github.com/jean-moreno/EdgeDetect-PostProcessingUnity/issues/6
I verified this quickly by copying and pasting the whole folder of shaders somewhere so I guess the path just needs changed again. Even then it didn't work in all modes. I think Before Transparent caused it to break fog rendering entirely but the other two options seemed to work ok. I didn't bother looking any further into it after that, there were some other issues with 2018.3 for me and I couldn't afford the time so I decided to delay upgrading for a bit until things support it better.
